docs/server-core/node/_sharedInstance.mdx

```diff
@@ -0,0 +1,26 @@
+```jsx
+// Create a shared instance that can be accessed globally
+const statsig = Statsig.newShared("secret-key");
+await statsig.initialize();
+
+// Access the shared instance from anywhere in your code
+const sharedStatsig = Statsig.shared();
+const isFeatureEnabled = sharedStatsig.checkGate(user, "feature_name");
+
+// Check if a shared instance exists
```

Comment:
Can you highlight that "hasSharedInstance" and "removeShared" may be useful to know about if you aren't sure if the shared instance is ready yet or if you want to switch to a new shared instance (removing the old one), but these aren't required in most uses cases where the shared instance is setup somewhere near the top of your application